app/views/cases/close.html.slim

```diff
@@ -18,6 +18,4 @@
 
 div class="case-#{@case.type_abbreviation.downcase}"
 
-  = render partial: "cases/#{@case.type_abbreviation.downcase}/close_form",
-           locals: { kase: @case,
-                     submit_button: t('.submit')}
+  = render partial: "cases/shared/date_responded_form", locals: { kase: @case, submit_button: t('.submit')}
```

Comment:
Probably need to remove `_close_form.html.slim`.

Reply:
I think this is used for the edit closure reasons form, I'll see how easy it is to do editting in this ticket but it might be worth splitting out this work